app/src/main/cpp/skyline/services/codec/IHardwareOpusDecoderManager.h

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <services/serviceman.h>
+
+namespace skyline::service::codec {
+    /**
+     * @brief Struct used to initialize a multi stream decoder
+     * @see opus_multistream_decoder_init()
+     */
+    struct MultiStreamParameters {
+        i32 sampleRate;
+        i32 channelCount;
+        i32 streams; //!< Number of streams
+        i32 coupledStreams; //!< Number of stereo streams
+        std::array<u8, 0x100> mappings; //!< Array of channel mappings
+    };
+    static_assert(sizeof(MultiStreamParameters) == 0x110);
+
+    /**
+     * @brief This service is used to get an instance of IHardwareOpusDecoder
```

Comment:
We generally discourage the use of `This X` for comments, don't comment on purely usage but more as to what's delegated to the class. I'd say something like `Manages all instances of IHardwareOpusDecoder`/`Manages creation of IHardwareOpusDecoder instances` would be more fitting.